packages/web3-eth-contract/package.json

```diff
@@ -49,6 +49,7 @@
 		"jest-extended": "^3.0.1",
 		"prettier": "^2.7.1",
 		"ts-jest": "^28.0.7",
-		"typescript": "^4.7.4"
+		"typescript": "^4.7.4",
+		"web3-eth-accounts": "^4.0.1-alpha.0"
```

Comment:
this change looks unneccesary

Reply:
Do you mean typescript or accounts change? For the former, a comma was added, and for the latter I need the package for testing.

Reply:
i am talking about `web3-eth-accounts` I think if its only for testing `web3-eth-accounts` should be under dev dependancies and it [seems like it is](https://github.com/web3/web3.js/blob/e8a831879d73024b3aa0ade39f3e93270582eb43/packages/web3-eth-contract/package.json#L53), or am i missing something

Reply:
oh never mind i get it, your right